### PR TITLE
Rachel PR

### PR DESCRIPTION
**Pull request by aagm**

### CODE REVIEW

File: app/views/shared/_sources_map.html.erb [hunk 1]
```diff
@@ -876,7 +877,7 @@
         <div class='source_table'>
           <dl class='sources_row'>
             <dt>Function</dt>
-            <dd>Identifies and locates the world’s Intact Forest Landscapes as of the year 2013 and changes in their extent from 2000 to 2013. Intact Forest Landscapes are undisturbed forest areas, large enough to retain all native biodiversity and showing no signs of human activity.</dd>
+            <dd>IIdentifies the world’s last remaining undisturbed forest areas, large enough to retain all native biodiversity and showing no signs of human activity as of the year 2013 and reduction in their extent from 2000-2013.</dd>
```
Comment (apercas) [inline app/views/shared/_sources_map.html.erb]: Fix the IIden... typo :)

**Comment by apercas** [comment]
Fix the typo and we'll merge it

**Comment by aagm** [comment]
:D done!

**Comment by apercas** [comment]
PADENTRO :dancer:

**Pull request merged**